app/src/controllers/plugins/uiExtensions/createImportProps.js

```diff
@@ -18,26 +18,47 @@ import React from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import moment from 'moment';
 import Parser from 'html-react-parser';
+import { reduxForm, formValueSelector } from 'redux-form';
 import { GhostButton } from 'components/buttons/ghostButton';
 import { BigButton } from 'components/buttons/bigButton';
 import { NavigationTabs } from 'components/main/navigationTabs';
 import { NoCasesBlock } from 'components/main/noCasesBlock';
 import { ItemList } from 'components/main/itemList';
-import { withModal } from 'controllers/modal';
+import { ModalLayout, ModalField } from 'components/main/modal';
+import { withModal, showModalAction } from 'controllers/modal';
```

Comment:
It seems that you don't need `withModal`, because you register modals in [plugin registraton function](https://github.com/reportportal/service-ui/pull/2345/files#diff-ad75650ec326305c792b348b0edec7b7R18)